### PR TITLE
Case 20877: Support CMake time configuration of QmlFileSelectors

### PR DESCRIPTION
**Pull request by danteruiz**
ticket - https://highfidelity.manuscript.com/f/cases/20877/Support-CMake-time-configuration-of-QmlFileSelectors

### CODE REVIEW

**Comment by danteruiz** [comment]
Test plan 
- run the android build of interface and make sure that the UI works currently.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17078/HighFidelity-Beta-PR14863-d0fecac.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17078/HighFidelity-Beta-PR14863-d0fecac.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17078/HighFidelity-Beta-Interface-PR14863-d0fecac.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17078/HighFidelity-Beta-PR14863-d0fecac.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17078/HighFidelity-Beta-Interface-PR14863-d0fecac.exe

File: libraries/shared/src/shared/FileUtils.cpp [hunk 1]
```diff
@@ -32,7 +32,7 @@ const QStringList& FileUtils::getFileSelectors() {
     std::call_once(once, [] {
 
 #if defined(Q_OS_ANDROID)
-        //extraSelectors << "android_" HIFI_ANDROID_APP;
+        extraSelectors << "android_" HIFI_ANDROID_APP;
```
Comment (NissimHadar) [inline libraries/shared/src/shared/FileUtils.cpp]: This _looks_ like a compilation error.

**Comment by JeffClinton** [comment]
Currently testing

File: libraries/shared/src/shared/FileUtils.cpp (hunk 1 above)
Comment (danteruiz) [reply libraries/shared/src/shared/FileUtils.cpp]: no it compiled fine, the HIFI_ANDROID_APP is defined in the cmake file

Comment (danteruiz) [reply libraries/shared/src/shared/FileUtils.cpp]: also jenkins produced builds

Comment (jherico) [reply libraries/shared/src/shared/FileUtils.cpp]: @NissimHadar the HIFI_ANDROID_APP expands to `"interface"` or `"questInterface"`.  It's perfectly legal in C++ to concatenate string literals by just putting them together `"like" "this"`

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17097/HighFidelity-Beta-PR14863-f07b5c8.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17097/HighFidelity-Beta-PR14863-f07b5c8.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17097/HighFidelity-Beta-Interface-PR14863-f07b5c8.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17097/HighFidelity-Beta-PR14863-f07b5c8.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17097/HighFidelity-Beta-Interface-PR14863-f07b5c8.exe

**Comment by JeffClinton** [comment]
Currently testing

**Pull request merged**